README.md

```diff
@@ -24,8 +24,8 @@ module "waf" {
 
   alb_arn                             = "${module.alb_web_containers.alb_arn}"
   associate_alb                       = true
-  ip_rate_limit                       = 2000
   ip_sets                             = [aws_wafregional_ipset.global.id]
+  rate_based_rules                    = [aws_wafregional_rate_based_rule.ipratelimit.id]
```

Comment:
can you include `aws_wafregional_rate_based_rule.ipratelimit` in this example so it's clear how it should be used?